setup.py

```diff
@@ -24,17 +24,14 @@
     requirements = ["numpydoc>=0.5"]
 else:
     requirements = [
-        'cameo>=0.7',
-        'sympy>=0.7.5',
-        'cobra>=0.4',
-        'ipython>=4.0',
-        'pyzmq>=14.5',
-        'numpy>=1.9.2',
-        'bokeh>=0.11.0',
-        'lxml>=3.4.4',
-        'six>=1.9.0',
-        'ipywidgets>=4.0',
-        'pandas>=0.16.2'
+        'cameo',
```

Comment:
No we can't because still using time-machine on ExpressionProfile which doesn't have a context manager..